src/app/login/login.component.ts

```diff
@@ -21,6 +21,7 @@ export class LoginComponent {
   returnUrl = this.route.snapshot.queryParams['returnUrl'] || '/';
   model = { name: '', password: '', repeatPassword: '' };
   message = '';
+  languageDirection: string = localStorage.getItem('direction');

```

Comment:
think this is not needed as direction is binding instead of languageDirection in login component.